test/sensors/camera/test_camera_model.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import torch
+
+from kornia.geometry.vector import Vector3
+from kornia.image import ImageSize
+from kornia.sensors.camera import CameraModel, CameraModelType
+from kornia.testing import BaseTester
+
+
+class TestPinholeCamera(BaseTester):
+    def _make_rand_data(self, batch_size, device, dtype):
+        params = torch.rand(batch_size, 4).to(dtype).to(device)
+        image_sizes = torch.randint(1, 100, (batch_size, 2)).to(dtype).to(device)
+        return params, ImageSize(image_sizes[:, 0], image_sizes[:, 1])
+
+    def test_smoke(self, device, dtype):
+        params, image_size = self._make_rand_data(1, device, dtype)
+        cam = CameraModel(image_size, CameraModelType.PINHOLE, params)
+        assert isinstance(cam, CameraModel)
+        self.assert_close(cam.params, params)
+        self.assert_close(image_size.height, cam.height)
+        self.assert_close(image_size.width, cam.width)
+
+    def test_cardinality(self, device, dtype):
+        pass
```

Comment:
can you skip the unnecessary tests too? 😅  makes a difference in the final number of the test suite

The JIT tests can be removed

```suggestion
    @pytest.mark.skip(reason='Unecessary test')
    def test_cardinality(self, device, dtype):
        pass
```